components/Tabs/index.js

```diff
@@ -7,3 +7,60 @@
 //
 //  The tab component should look like this:
 //    <div class="tab">topic here</div>
+
+// select the main dom node to attach our dynamic content
+
+const tabs = document.querySelector(".topics");
+
+// send GET request using axios
+
+axios
+  .get(`https://lambda-times-backend.herokuapp.com/topics`)
+
+  .then(resolved => {
+    const apiData = resolved.data;
+    // Handles Success: here's where we get the results from server
+
+    //console.log("Handles success", apiData);
+
+    const topics = apiData["topics"];
+
+    //console.log("topics", topics);
+
+    // Using forEach to iterate over the topics creating a new Tab component and add it to the DOM
+    // Use forEach to iterate over array topics
+    topics.forEach(topic => {
+      // Create new tab component in reference to function being created below 
+      newTab = createTabs(topic);
+      // Add to DOM
+      tabs.appendChild(newTab);
```

Comment:
Here in this axios `.then` you could put all of that together to write it in a more compact way like so ...

```
 .then(resolved => {
    resolved.data.topics.forEach(topic => {
      tabs.appendChild(Tab(topic));
    });
  })
```